dump/requirements.txt

```diff
@@ -0,0 +1,4 @@
+blessings
+boto3
+click
+requests
```

Comment:
Missing `cnx-common` in this list

Reply:
O... my. The sloppiness of all this is terrible. 😝